common/v2/features/PurchaseMembership/config.ts

```diff
@@ -60,55 +60,54 @@ export const MEMBERSHIP_CONFIG: IMembershipConfigObject = {
   },
 
   threemonths: {
-    title: '3 months',
-    name: 'Three Month Membership',
+    title: translateRaw('MEMBERSHIP_MONTH', { $duration: '3' }),
     key: IMembershipId.threemonths,
     contractAddress: '0xae90e0F7F3f2191B17b1816dFA8C5Ce8e049DC96',
     description: '',
     icon: threemonthsIcon,
     price: '10.5',
+    discount: '10',
     assetUUID: DAIUUID,
     durationInDays: 90,
-    discountNotice: '10% off'
+    discountNotice: translateRaw('MEMBERSHIP_DISCOUNT', { $percentage: '10%' })
   },
 
   sixmonths: {
-    title: '6 months',
-    name: 'Six Month Membership',
+    title: translateRaw('MEMBERSHIP_MONTH', { $duration: '6' }),
     key: IMembershipId.sixmonths,
     contractAddress: '0x1C8369C9772E71E82679c9750E4770F29FECcbdD',
     description: '',
     icon: sixMonthsIcon,
     price: '18',
+    discount: '20',
     assetUUID: DAIUUID,
     durationInDays: 180,
-    discountNotice: '20% off'
+    discountNotice: translateRaw('MEMBERSHIP_DISCOUNT', { $percentage: '20%' })
   },
 
   twelvemonths: {
-    title: '12 months',
-    name: 'Twelve Month Membership',
+    title: translateRaw('MEMBERSHIP_MONTH', { $duration: '12' }),
     key: IMembershipId.twelvemonths,
     contractAddress: '0xb9cae1F4480bcc6dF52F045077BAc1DE448D5406',
     description: '',
     icon: twelveMonthsIcon,
     price: '30',
+    discount: '40',
     assetUUID: DAIUUID,
     durationInDays: 366,
-    discountNotice: '40% off'
+    discountNotice: translateRaw('MEMBERSHIP_DISCOUNT', { $percentage: '40%' })
```

Comment:
Can we use the properties from the config in the education panel such that the text is defined in one place only? 😄